uws/UWS/models.py

```diff
@@ -10,6 +10,25 @@ def _parse_bool(self, value):
             return False
         return value
 
+    def _find_ns_uws(self, namespaces):
+        # Search for namespace that contains "http://www.ivoa.net/xml/UWS/"
+        # (usually, there is a version attached, which we ignore here),
+        # for using it (including {}) as uws_ns in the
+        # find/findall-functions everywhere below.
+        ns = None
+        for p in namespaces.keys():
+            value = namespaces[p]
+            if 'http://www.ivoa.net/xml/UWS/' in value:
+                ns = value
+
+        if ns is None:
```

Comment:
if you return above and end up here, you can just raise an error:

`raise UWSError("no matching UWS namespace found in xml-response, probably won't find anything.")`

or just print the warning. But I think we can be strict here and just say, this is not UWS, I have no clue what I am doing, so just raise and die.